client/login/magic-login/handle-emailed-link-form.jsx

```diff
@@ -159,19 +159,17 @@ class HandleEmailedLinkForm extends React.Component {
 			<Button primary disabled={ this.state.hasSubmitted } onClick={ this.handleSubmit }>
 				{ this.props.isImmediateLoginAttempt
 					? translate( 'Confirm Login' )
-					: translate( 'Finish Login' ) }
+					: translate( 'Continue to WordPress.com' ) }
 			</Button>
 		);
 
 		let title;
 		if ( this.props.isManualRenewalImmediateLoginAttempt ) {
-			title = translate(
-				'Continue to WordPress.com to update your payment details and renew your subscription'
-			);
+			title = translate( 'Update your payment details and renew your subscription' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Log in to update your payment details and renew your subscription' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/handleemailedlinkform-continue-button-label)